src/pk_core.c

```diff
@@ -613,7 +617,9 @@ DEF(coreStrChr,
   int64_t num;
   if (!validateInteger(vm, ARG(1), &num, "Argument 1")) return;
 
-  // TODO: validate num is a byte.
+  if (!isAByte(num)) {
+    RET_ERR(newString(vm, "The number is greater than a byte."));
```

Comment:
The number could also be lesser than a byte range, change this error message to something like `The number is not in a byte range.`